sentry/values.yaml

```diff
@@ -456,6 +459,8 @@ hooks:
     affinity: {}
     nodeSelector: {}
     # tolerations: []
+  snubaMigrate:
```

Comment:
Don't you need to use `snubaMigrate: {}`?

Reply:
> Don't you need to use `snubaMigrate: {}`?

Yes, I have added it. Thank You :)